src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java

```diff
@@ -115,12 +115,10 @@ private void processQuery(JobContext context) throws IOException, InterruptedExc
       TableReference exportTableReference = createExportTableReference(type, datasetProjectId, datasetId,
                                                                        temporaryTableName, configuration);
       runQuery(configuration, bigQueryHelper, projectId, exportTableReference, query, location);
-      if (type == Type.VIEW || type == Type.MATERIALIZED_VIEW) {
-        configuration.set(BigQueryConfiguration.INPUT_PROJECT_ID_KEY,
-                          configuration.get(BigQueryConstants.CONFIG_VIEW_MATERIALIZATION_PROJECT));
-        configuration.set(BigQueryConfiguration.INPUT_DATASET_ID_KEY,
-                          configuration.get(BigQueryConstants.CONFIG_VIEW_MATERIALIZATION_DATASET));
-      }
+      configuration.set(BigQueryConfiguration.INPUT_PROJECT_ID_KEY,
```

Comment:
nit: Add a comment saying that these values come from the default sink config and can be overridden via config

Reply:
done